src/configure

```diff
@@ -27,7 +27,7 @@ CONFIGURE_VERSION=2
 
 function rel2abs {
   if [ ! -z "$1" ]; then
-    local retval=`cd $1 2>/dev/null && pwd || exit 1`
```

Comment:
OK -- just print some informative message (or you can try create the
directory and call the read_dirname again)?
y.

On Tue, Oct 20, 2015 at 9:42 AM, dbdean <notifications@github.com> wrote:

> In src/configure
> <https://github.com/kaldi-asr/kaldi/pull/278#discussion_r42495940>:
>
> > @@ -27,7 +27,7 @@ CONFIGURE_VERSION=2
> >
> >  function rel2abs {
> >    if [ ! -z "$1" ]; then
> > -    local retval=`cd $1 2>/dev/null && pwd || exit 1`
>
> The intention was to pass a directory to configure using --prefix=PREFIX,
> but that the directory may not already exist. I used the existing
> read_dirname function so as not to duplicate functionality, but discovered
> that it doesn't return the correct path if it doesn't already exist (which
> readlink -f) does.
>
> Perhaps it would be ok if the prefix had to already exist, and then we
> don't have to make any changes to this function.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/278/files#r42495940>.
>